lib/base64.nit

```diff
@@ -84,14 +86,16 @@ redef class String
 		return result.to_s_with_length(result_length)
 	end
 
-	# Decodes the receiver string from base64.
-	# By default, uses "=" for padding.
-	fun decode_base64 : String do return decode_base64_custom_padding('='.ascii.to_b)
-
-	# Decodes the receiver string to base64 using a custom padding character.
+	# Decodes the receiver string to base64.
 	#
-	# If using the default padding character `=`, see `decode_base64`.
-	fun decode_base64_custom_padding(padding : Byte) : String
+	# If no padding is given, will use `=` as the padding character
+	#
+	#     assert "".decode_base64 == ""
+	#     assert "TW9ycmlhciBybHkgbG92ZXMgZG9jcw==".decode_base64 == "Morriar rly loves docs"
+	#     assert "TW9ycmlhciBybHkgbG92ZXMgZG9jcw--".decode_base64(45.to_b) == "Morriar rly loves docs"
```

Comment:
The examples and the doc should state what happens when you give something else than a base64 to decode.